akvo/templates/myrsr/project_editor/section_1.html

```diff
@@ -86,7 +86,7 @@ <h5>{% trans 'Project photo' %}</h5>
                 {% if custom_fields_section_1 %}
                     <div class="row">
                         <div class="col-md-12">
-                          <h4>Additional fields for {{ project.primary_organisation }}</h4>
+                          <h4>{% trans "Additional fields for" %} {{ project.primary_organisation }}</h4>
```

Comment:
Doing this using blocktrans allows the translator to put the variable in the correct place if the language has another ordering of words:

```
{% blocktrans with primary_organisation_name=project.primary_organisation %}
    Additional fields for {{ primary_organisation_name }}
{% endblocktrans %}
```